app/ux/WebView.js

```diff
@@ -412,7 +412,7 @@ Ext.define('Rambox.ux.WebView',{
 	,setUnreadCount: function(newUnreadCount) {
 		var me = this;
 
-		if (me.record.get('includeInGlobalUnreadCounter') === true) {
+		if (newUnreadCount === parseInt(newUnreadCount,10) && me.record.get('includeInGlobalUnreadCounter') === true) {
```

Comment:
Actually, this should be `if ( !isNaN(newUnreadCount) && (function(x) { return (x | 0) === x; })(parseFloat(newUnreadCount)) && me.record.get('includeInGlobalUnreadCounter') === true)`

(I'm not a git expert, so I'm not sure how to best update my pull request)